src/core/TimeDelta.cpp

```diff
@@ -82,3 +83,10 @@ int TimeDelta::getYears() const
 {
     return m_years;
 }
+
+int TimeDelta::toDays() const
+{
+    QDateTime now = Clock::currentDateTime();
+    QDateTime future = now + *this;
+    return now.daysTo(future);
+}
```

Comment:
This function is out of place in this class. It implies a comparison from current time, which none of the other calls make. This whole class seems extremely pointless to me, to be honest.